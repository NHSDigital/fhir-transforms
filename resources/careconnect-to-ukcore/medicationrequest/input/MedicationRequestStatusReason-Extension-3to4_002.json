{
  "resourceType": "MedicationRequest",
  "extension": [
    {
      "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-MedicationStatusReason-1",
      "extension": [
        {
          "url": "statusReason",
          "valueCodeableConcept": {
            "text": "Alergy"
          }
        },
        {
          "url": "statusChangeDate",
          "valueDateTime": "2010-08-09T14:27:43.233+01:00"
        }
      ]
    }
  ]
}
