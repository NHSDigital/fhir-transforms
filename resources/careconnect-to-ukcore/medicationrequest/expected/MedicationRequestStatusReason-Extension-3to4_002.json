{
  "resourceType": "MedicationRequest",
  "statusReason": {
    "text": "Alergy"
  }
}
